pkgs/development/python-modules/sqlalchemy/default.nix

```diff
@@ -23,7 +23,7 @@ buildPythonPackage rec {
   '';
 
   checkPhase = ''
-    pytest test
+    pytest test -k 'not exception_persistent_flush_py3k'
```

Comment:
If we condition this on `isPy35` then it won't rebuild much.

Reply:
this is very true... I'll edit it